modules/virt-update-node-network-config-form.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: PROCEDURE
+[id="virt-update-node-network-config-form_{context}"]
+= Updating the policy using form
+
+.Procedure
+. Navigate to *Networking* → *NodeNetworkConfigurationPolicy*.
+. In the *NodeNetworkConfigurationPolicy* page, click on the kebab icon (three dots) within the row corresponding to the policy you wish to edit, and click *Edit*.
+. Edit the fields that you might want to update.
++
+[NOTE]
+====
+You can update the read-only fields in the *Edit NodeNetworkConfigurationPolicy* page using YAML.
```

Comment:
We should not advise something like that. The read-only fields are read-only because changing those fields could generate some kind of error. 
For example, renaming an interface would not actually rename the interface in the backend. It would create a new one (if the backend does not throw an error)

Reply:
I see. So basically, we are not going to 'reveal' at this point that they could go to YAML and do the changes.
I will remove the note.

Reply:
@upalatucci Can you verify if this looks fine now? If yes, can you provide an approval?